SecretSanta/src/SecretSanta.Api/Controllers/PairingController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using SecretSanta.Domain.Services.Interfaces;
+
+namespace SecretSanta.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class PairingController : ControllerBase
+    {
+        private IPairingService PairingService {get;set;}
+        private IMapper Mapper { get; set; }
```

Comment:
Because these properties are only set int he constructor they can be made readonly.
```suggestion
        private IMapper Mapper { get; }
```